debian/utils/python

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/python3 "$@"
```

Comment:
We need to make sure that only python3 and its related tools are invoked by setup.py. The same result might be achievable by patching any instance of  "!/usr/bin/env python" to "!/usr/bin/env python3" in the upstream source, but I haven't tried this.